File: BmiCalc/app/src/main/java/pamo/bmicalc/ui/quiz/QuizQuestion.java
package pamo.bmicalc.ui.quiz;

import java.util.List;

public class QuizQuestion {
    private final int questionId;
    private final String questionText;
    private final List<QuizAnswer> answers;

    public QuizQuestion(int question_id, String question_text, List<QuizAnswer> answers) {
        this.questionId = question_id;
        this.questionText = question_text;
        this.answers = answers;
    }

    public int getQuestionId() {
        return questionId;
    }

    public String getQuestionText() {
        return questionText;
    }

    public List<QuizAnswer> getAnswers() {
        return answers;
    }
}

File: BmiCalcKotlin/app/src/main/java/pamo/bmicalc_kotlin/ui/quiz/QuizQuestion.kt
package pamo.bmicalc_kotlin.ui.quiz

import java.util.*

class QuizQuestion(var questionId: Int, var questionText: String, answers: MutableList<QuizAnswer?>) {
    private var answers: MutableList<QuizAnswer?> = ArrayList()

    init {
        this.answers = answers
    }

    fun getAnswers(): MutableList<QuizAnswer?> {
        return this.answers
    }
}